examples/demo_lfr.ipynb

```diff
@@ -4,6 +4,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://github.com/Trusted-AI/AIF360/blob/master/examples/demo_lfr.ipynb)\n",
+    "\n",
```

Comment:
Why is "\n" added here but not there in above two links?

Reply:
Because for the first two, I added a new markdown cell to put the colab link.
And in the last one, there was already a markdown cell where I just appended the colab link.